CHANGELOG.md

```diff
@@ -16,3 +16,11 @@ Pre-2.2 the HTTP Routing Name is configurable via the `http.routing.name` option
 2. Add this parameter to a **single** profile in the affected CDN
 
 With those profile parameters in place Traffic Ops can be safely upgraded to 2.2. Before taking a post-upgrade snapshot, make sure to check your Delivery Service example URLs for unexpected Routing Name changes. Once Traffic Ops has been upgraded to 2.2 and a post-upgrade snapshot has been taken, your Traffic Routers can be upgraded to 2.2 (Traffic Routers must be upgraded *after* Traffic Ops so that they can work with custom per-DeliveryService Routing Names).
+
+
+#### Backup Edge Cache group
```

Comment:
since we currently have an RC for 2.2, this is most likely landing in 2.3. So I would add a new v2.3.0 [unreleased] section at the top of this file and add this subsection `#### Backup Edge Cache group` under a `### Release Notes` header, ie:
```
v2.3.0 [unreleased]
-------------------

### Release Notes

#### my new feature
...

v2.2.0 [unreleased]
...
```